classes/item.rb

```diff
@@ -0,0 +1,40 @@
+require 'date'
+
+class Item
+  attr_writer :genre, :author, :source, :lable
+  attr_reader :id, :archived
+  attr_accessor :publish_date
+
+  def initialize(publish_date, archived: false)
+    super()
+    @id = Random.rand(1...1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def add_genre(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
+
+  def add_label(lable)
+    @lable = lable
+  end
```

Comment:
- [ ] Item class has one Genre, Author, and Label. Kindly make sure you correctly implement the add_label method as you did with the add_genre and add_author. Your code should look like this:

```ruby
  def label=(label)
      @label = label
      label.items.push(self) unless label.items.include?(self)
    end
```